NuRadioMC/test/Veff/1e18eV/T03check_output.py

```diff
@@ -11,24 +11,21 @@
 
 # the event generation has a fixed seed and I switched to Alvarez2000 (also no randomness)
 # thus, the Veff has no statistical scatter
-Veff_mean = 5.45361
```

Comment:
How did this value change?

Reply:
see https://github.com/nu-radio/NuRadioMC/commit/fec51da0f0998bbb5dfa2d05a46cbe9f63ea5c01#diff-6ee18f45f38c958db1d8604849a200ee
It only changed within the statistical precision. However we test much more stringent which is possible because we have a fixed input file. Thus the same events should always trigger. The new structure (one line per shower instead of event) required a new input file which solved the random distribution of the events. 
The story is even longer (and should be documented by commit messages). I was able to reproduce the exact same Veff with the new structure. But then I needed to change the order in which the different variables were generated in the event generator. I think before it was first zenith angles and then vertex positions. Now it is the other way around which changed the random distribution of the events.